NethServer/Firewall.pm

```diff
@@ -451,6 +452,24 @@ sub getAliasZone($)
     }
 }
 
+=head2 isDanglingZoneRecord(zone)
```

Comment:
We could encapsulate the isDanglingZoneRecord integrity check also inside the NetworksDB::zones() method implementation. It could be more subject to regressions though.